ocaml/idl/datamodel.ml

```diff
@@ -3723,6 +3723,17 @@ let task_destroy = call ~flags:[`Session]
   ~params:[Ref _task, "self", "Reference to the task object"]
   ~allowed_roles:_R_READ_ONLY (* POOL_OP can destroy any tasks, others can destroy only owned tasks *)
   ()
+
+let task_set_status = call ~flags:[`Session]
+  ~in_oss_since:None
+  ~in_product_since:rel_dundee
+  ~name:"set_status"
+  ~doc:"Set the task status"
+  ~params:[Ref _task, "self", "Reference to the task object";
+   status_type, "status", "task status"]
```

Comment:
The standard setters normally call the parameter for the value to-be-set `value`.